src/gatling/simulations/uk/gov/hmcts/jui/sscs/scenario/JUILogin.scala

```diff
@@ -13,28 +13,32 @@ object JUILogin {
   val MinThinkTime = Environment.minThinkTime
   val MaxThinkTime = Environment.maxThinkTime
   val JUIBaseUrl = scala.util.Properties.envOrElse("URL_TO_TEST", Environment.URL_TO_TEST).toLowerCase()
-
-  val submitLogin = 	exec(http("TC02_JUI_SubmitLogin")
-    .post(IdamJUIURL + "/login?response_type=code&client_id=juiwebapp&redirect_uri=" + JUIBaseUrl + "/oauth2/callback").disableFollowRedirect
-    .formParam("username", "${SSCSUserName}")
-    .formParam("password", "${SSCSUsrPwd}")
-    .formParam("continue", JUIBaseUrl + "/oauth2/callback")
-    .formParam("upliftToken", "")
-    .formParam("response_type", "code")
-    .formParam("_csrf", "${csrftoken}")
-    .formParam("redirect_uri", JUIBaseUrl + "/oauth2/callback")
-    .formParam("client_id", "juiwebapp")
-    .formParam("scope", "")
-    .formParam("state", "")
-    .check(headerRegex("Location", "(?<=code=)(.*)").saveAs("authCode"))
-    .check(status.in(200,302)))
+  val commonSSCSHeader = Environment.commonSSCSHeader
+  val headers_0 = Environment.headers_0
+  val headers_1 = Environment.headers_1
+  val headers_3 = Environment.headers_3
+
+  val submitLogin = exec(http("TX02_JUI_SubmitLogin_01")
+    .post(IdamJUIURL + "/login?response_type=code&client_id=juiwebapp&redirect_uri=" + JUIBaseUrl + "/oauth2/callback")//.disableFollowRedirect
+    .headers(headers_0)
+    .formParam("username", "juitestjudgesscs@mailnesia.com")
```

Comment:
wouldn't this username and password should be coming from vault

Reply:
Oops yes, will amend

Reply:
@satyachundur please could you re-review the change and approve?